cmd/zed/import.go

```diff
@@ -140,13 +142,34 @@ func importRelationships(client *authzed.Client, relationships string) error {
 	if err := scanner.Err(); err != nil {
 		return err
 	}
+	log.Info().Int("count", len(relationshipUpdates)).Msg("parsed relationships")
 
-	request := &v1.WriteRelationshipsRequest{Updates: relationshipUpdates}
-	log.Trace().Interface("request", request).Msg("writing relationships")
-	log.Info().Int("count", len(relationshipUpdates)).Msg("importing relationships")
+	return batchRequests(client, relationshipUpdates, importBatchSize)
+}
 
-	if _, err := client.WriteRelationships(context.Background(), request); err != nil {
-		return err
+func batchRequests(client *authzed.Client, updates []*v1.RelationshipUpdate, batchSize int) error {
+	totalBatches := (len(updates) + batchSize - 1) / batchSize
+	log.Info().Int("batch_size", batchSize).Int("total_batches", totalBatches).Msg("batching relationship writes")
+
+	for i := 0; i < totalBatches; i++ {
```

Comment:
@ecordell Had a review (that's gone now? Maybe a ninja delete?) about the loop potentially being simpler. I re-tooled the loop to make it hopefully more straightforward.